lib/middleware/runner.js

```diff
@@ -1,3 +1,4 @@
+// @ts-check
```

Comment:
?

Reply:
Sorry, the `@ts-check` helped me to write the PR, forgot to remove it again.